### DISCUSSION TITLE
https://plugin-lightgallery.vuejs.press/zh/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#954] [Announcements]
# https://plugin-lightgallery.vuejs.press/zh/demo.html

为了让你更好的了解如何使用插件，我们使用 vuepress-plugin-lightgallery 和 @vuepress/theme-default 制作了一个案例。 源代码 预览

https://plugin-lightgallery.vuejs.press/zh/demo.html

**Comment by sweetmei** [comment]
vnnvn
